### DISCUSSION TITLE
Change virtual osd keyboard

### DISCUSSION THREAD

**Discussion by MrMeatballSub** [jurialmunkey/skin.arctic.horizon.2#888] [Discussion]
Is it possible to revert back to the original kodi on screen virtual keyboard?  Just personal preference.

Also, while using the search, is it possible to save search history and have it appear somewhere under the search bar or something?  Or for the search bar to remember the last thing you typed in even when switching between different sections, playing movies, etc.

Thanks the skin is amazing

**Comment by DROIDshield** [comment]
Love the idea about the search saving it's history. Although I do use speech to search so that makes life easier but a history would be good 👍
